opm/test_util/summaryComparator.cpp

```diff
@@ -204,7 +206,71 @@ const char* SummaryComparator::getUnit(const char* keyword){
 }
 
 
+void SummaryComparator::printKeywords(){
+    size_t ivar = 0;
+    std::vector<std::string> noMatchString; 
+    std::cout << "Keywords that are common for the files:" << std::endl;
+	while(ivar < stringlist_get_size(keysLong)){
+	    const char* keyword = stringlist_iget(keysLong, ivar);
+	    if (stringlist_contains(keysLong, keyword) && stringlist_contains(keysShort, keyword)){
+		std::cout << keyword << std::endl; 
+		ivar++;
+	    }
+	    else{
+		noMatchString.push_back(keyword);
+		ivar++;
+	    }
+	}
+    if(noMatchString.size() == 0){
+	std::cout << "No keywords were different" << std::endl; 
```

Comment:
be consistent with the space/tab indentation.  I believe OPM prefers spaces, but you should definitely not mix spaces and tabs in a file (let alone inside one function).